cdap-ui/app/features/admin/templates/partials/app-dd.html

```diff
@@ -1,20 +1,21 @@
-<ul class="dropdown-menu" role="menu">
+<!-- <ul class="dropdown-menu dropdown-menu-right" role="menu">
```

Comment:
Just a minor comment but if we are not going to use the commented out code can we remove them? For some reason its a little distracting.